### ISSUE TITLE
"Pick organization..." should be an <h4>

### ISSUE THREAD

**Issue by msecret** [18F/cg-dashboard#362]
```
The heading structure is not logically nested. This h5 element should be an h4 to be properly nested.
```

**Comment by msecret** [comment]
How important is this in terms of accessibility? I feel like this doesn't really make much sense to be a `h4`.
